docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[host-risk-score]]
+== Host Risk Score metadata
+
+The Host Risk Score feature highlights risky hosts from within your environment. It utilizes a transform with a scripted metric aggregation to calculate host risk scores based on detection rule alerts with an "open" status within a 5-day window. The transform runs hourly to keep the score updated as new detection rule alerts are generated.
+
+The contribution of each rule to the host risk score is calculated as a function of the rule risk score (`signal.rule.risk_score`) and a time decay factor to reduce the contribution of stale alerts, while considering recent alerts. A weighted sum of the individual time-corrected rule risks is then applied to the risk score. As such, rules with a higher time-corrected risk also have a higher weight. Each host risk score is on a scale of 0 to 100.
+
+The following table shows how risk levels are applied to a host, based on the normalized risk score:
+
+[width="100%",options="header"]
+|==============================================
+|Risk level |Host Risk Score
+
+|Unknown |< 20
+|Low |20-40
+|Moderate |40-70
+|High     | 70-90
+|Critical  | > 90
+
+
+|==============================================
+
+[discrete]
+=== Deploy the Host Risk Score package
+
+To deploy the Host Risk Score framework in your environment, follow the steps outlined in the https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md[readme file]. These instructions also include steps to enable the `riskyHostsEnabled` feature flag.
+
+NOTE: To view host risk score data on the {elastic-sec} Overview page and the Alert details flyout, you must enable the feature flag. However, enabling the feature flag is NOT required to view the dashboards.
+
+[discrete]
+=== View host risk score data
+
+If the `riskyHostsEnabled` feature flag is enabled:
+
+. Go to the {security-app}.
+. On the Overview page, locate the *Current host risk scores* card in the lower-right corner.
+. Click *View dashboard*.
+
+[role="screenshot"]
+image::images/host-score-overview.png[]
+
+If the `riskyHostsEnabled` feature flag is NOT enabled:
+
+. In {kib}, go to *Analytics -> Dashboard*.
+. Select one of the following Host Risk Score dashboards:
+* Current Risk Score for Hosts
+* Current Risk Score for Hosts v2
+* Drilldown of Host Risk Score
+* Host Risk Score v2
++
+[role="screenshot"]
+image::images/select-dashboard.png[]
+. In the *Current Risk Scores for Hosts* list, hover your cursor over the host name to view, click the *+* button, then select *Go to Dashboard*.
```

Comment:
I think a word might be missing after "view". Is that word "menu" or does the **+** button appear?

Reply:
@nastasha-solomon the `+` button appears as an inline action once you hover over a host name.